app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md

```diff
@@ -16,14 +18,24 @@ expander:
     title: teacher degree apprenticeships
     text: "You can normally only do a teacher degree apprenticeship (TDA) if you've been a resident in the UK for the last 3 years or more. There are some exceptions to this. For example, if you've applied to the Afghan or Ukraine resettlement schemes. Speak to providers for more information."
 ---
-If you do not already have an undergraduate degree, you may be eligible to do a teacher degree apprenticeship (TDA), a new route to become a qualified teacher.
+If you do not already have an undergraduate degree, you may be eligible to do a teacher degree apprenticeship (TDA). 
+
+The TDA is a teacher training programme which offers a new route to become a qualified teacher. 
+
+The [teacher training providers confirmed so far offer TDAs in secondary maths](https://www.gov.uk/government/publications/secondary-maths-teacher-degree-apprenticeship-funding-pilot-providers/providers-selected-to-deliver-the-teacher-degree-apprenticeship-tda-funding-pilot). TDAs in other subjects will be added throughout the year, up until September 2025, with training due to start in autumn 2025. 
+
+[View the teacher degree apprenticeship courses currently available]( https://find-teacher-training-courses.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&university_degree_status=false&visa_status=false).
```

Comment:
Could this be shortened to TDA? We use TDA in the link above so it threw me off slightly that we go from acronym back to full title?

Reply:
I think this is probably better in full - TDA is very much internal language at present